src/storage/disk_manager.c

```diff
@@ -832,21 +832,22 @@ disk_unformat (THREAD_ENTRY * thread_p, const char *vol_fullname)
  *   return: NO_ERROR
  *   volid(in): Volume identifier
  *   new_vol_fullname(in): New volume label/name
- *   new_dbcreation(in): New database creation time
+ *   new_db_creation(in): New database creation time
  *   new_chkptlsa(in): New checkpoint
  *   logchange(in): Whether or not to log the change
  *   flush_page(in): true for flush dirty page. otherwise, false
  *
  * Note: This function is targeted for the log and recovery manager. It is used when a database is copied or renamed.
  */
 int
-disk_set_creation (THREAD_ENTRY * thread_p, INT16 volid, const char *new_vol_fullname, const INT64 * new_dbcreation,
+disk_set_creation (THREAD_ENTRY * thread_p, INT16 volid, const char *new_vol_fullname, const INT64 * new_db_creation,
 		   const LOG_LSA * new_chkptlsa, bool logchange, DISK_FLUSH_TYPE flush)
 {
   DISK_VOLUME_HEADER *vhdr = NULL;
   LOG_DATA_ADDR addr = LOG_DATA_ADDR_INITIALIZER;
   DISK_RECV_CHANGE_CREATION *undo_recv;
   DISK_RECV_CHANGE_CREATION *redo_recv;
+  bool is_new_db_create = false;
```

Comment:
```suggestion
  bool is_db_created = false;
```
또는 is_new_db_created 
정도로 하면 어떨까요?

Reply:
제거하는 방향으로 변경하였습니다.